test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -300,3 +302,62 @@ def test_snapshot_without_logs(self):
         # Cleanup
         self.du.rm(path=snap_dir, recursive=True, force=True)
         self.du.rm(path=output_tar, sudo=True, force=True)
+
+    def test_obfuscate_resv_user_groups(self):
+        """
+        Test obfuscation of user & group related attributes while capturing
+        snapshots via pbs_snapshot
+        """
+        now = int(time.time())
+
+        # Let's submit a reservation with Authorized_Users and
+        # Authorized_Groups set
+        attribs = {ATTR_auth_u: TEST_USER1, ATTR_auth_g: TSTGRP0,
+                   ATTR_l + ".ncpus": 2, 'reserve_start': now + 25,
+                   'reserve_end': now + 45}
+        resv_obj = Reservation(attrs=attribs)
+        resv_id = self.server.submit(resv_obj)
+        attribs = {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}
+        self.server.expect(RESV, attribs, id=resv_id)
+
+        # Now, take a snapshot with --obfuscate
+        target_dir = os.getcwd()
```

Comment:
can this be tmp which in most cases is likely to pass for any user that is going to run this PTL script.